### PR TITLE
Skip checking delete for default NEG

### PR DESCRIPTION
**Pull request by skmatti**
/assign @freehan

### CODE REVIEW

File: pkg/fuzz/gcp.go [hunk 1]
```diff
@@ -237,12 +239,17 @@ func (g *GCLB) CheckResourceDeletion(ctx context.Context, c cloud.Cloud, options
...
-		_, err := c.BetaNetworkEndpointGroups().Get(ctx, &k)
+		ns, err := c.BetaNetworkEndpointGroups().Get(ctx, &k)
 		if err != nil {
 			if err.(*googleapi.Error) == nil || err.(*googleapi.Error).Code != http.StatusNotFound {
 				return fmt.Errorf("NetworkEndpointGroup %s is not deleted/error to get: %s", k.Name, err)
 			}
 		} else {
+			// TODO(smatti): Add NEG description to make this less error prone.
+			if options != nil && options.SkipDefaultBackend &&
+				strings.Contains(ns.Name, fmt.Sprintf("%s-%s", kubeSystemNS, defaultHTTPBackend)) {
```
Comment (freehan) [inline pkg/fuzz/gcp.go]: are you sure the name is not truncated?

Comment (skmatti) [reply pkg/fuzz/gcp.go]: I think default http name and namespace are constant values mentioned above. The only fields that lead to truncation are these two fields itself for NEG.

Comment (freehan) [reply pkg/fuzz/gcp.go]: Just add comment about the limitations here

If the default backend service name changed, it may not get recognized here.

Comment (freehan) [reply pkg/fuzz/gcp.go]: This is to prevent L7 ILB tests (using NEG backend service) to block other tests on WaitForDeletion..

**Pull request merged**